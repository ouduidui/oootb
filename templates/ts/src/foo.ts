export const saySomething = (msg: string) => `hello, ${msg}`
