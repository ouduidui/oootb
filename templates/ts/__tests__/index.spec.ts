import { expect, it } from 'vitest'
import { saySomething } from '~/foo'

it('happy path', () => {
  expect(saySomething('world')).toMatchSnapshot('hello, world')
})
