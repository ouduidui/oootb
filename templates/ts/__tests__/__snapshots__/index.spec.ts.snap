// Vitest Snapshot v1

exports[`happy path > hello, world 1`] = `"hello, world"`;
